<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QToolBox" name="remoteControlToolBox">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="serverPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>751</width>
         <height>544</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Server</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="connectionGroupBox">
          <property name="title">
           <string>Connection</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="addressLabel">
             <property name="text">
              <string>Address</string>
             </property>
             <property name="buddy">
              <cstring>addressLineEdit</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="addressLineEdit">
             <property name="text">
              <string>localhost</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="portLabel">
             <property name="text">
              <string>Port</string>
             </property>
             <property name="buddy">
              <cstring>portSpinBox</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="portSpinBox">
             <property name="maximum">
              <number>65535</number>
             </property>
             <property name="value">
              <number>5572</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="authenticationGroupBox">
          <property name="title">
           <string>Authentication</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="realmLabel">
             <property name="text">
              <string>Realm</string>
             </property>
             <property name="buddy">
              <cstring>realmLineEdit</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="realmLineEdit">
             <property name="text">
              <string>rclone</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="userLabel">
             <property name="text">
              <string>User</string>
             </property>
             <property name="buddy">
              <cstring>userLineEdit</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="userLineEdit"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="passwordLabel">
             <property name="text">
              <string>Password</string>
             </property>
             <property name="buddy">
              <cstring>passwordLineEdit</cstring>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="passwordLineEdit">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="hostGroupBox">
          <property name="title">
           <string>Host</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QWidget" name="hostActionWidget" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="hostActionPushButton">
                <property name="text">
                 <string>Start</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="hostLogPlainTextEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="serverStatusWidget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QWidget" name="serverStatusWidget_2" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="serverStatusPushButton">
                <property name="text">
                 <string>Check</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="serverStatusSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="serverLogPlainTextEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="serverSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="remotesPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>765</width>
         <height>440</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Remotes</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWidget" name="remotesActionWidget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="remotesConnectionPushButton">
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="remotesAddPushButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="remotesRemovePushButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="remotesBrowsePushButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Browse</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="remotesSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="remoteListWidget"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="browsePage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>139</width>
         <height>162</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Browse</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QWidget" name="commandWidget" native="true">
          <layout class="QFormLayout" name="formLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="sourceLabel">
             <property name="text">
              <string>Source</string>
             </property>
             <property name="buddy">
              <cstring>sourceLineEdit</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="sourceLineEdit"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="destinationLabel">
             <property name="text">
              <string>Destination</string>
             </property>
             <property name="buddy">
              <cstring>destinationLineEdit</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="destinationLineEdit"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="browseTreeWidget">
          <column>
           <property name="text">
            <string>Path</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Source</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Destination</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Action</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>783</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <tabstops>
  <tabstop>connectionGroupBox</tabstop>
  <tabstop>addressLineEdit</tabstop>
  <tabstop>portSpinBox</tabstop>
  <tabstop>authenticationGroupBox</tabstop>
  <tabstop>realmLineEdit</tabstop>
  <tabstop>userLineEdit</tabstop>
  <tabstop>passwordLineEdit</tabstop>
  <tabstop>hostGroupBox</tabstop>
  <tabstop>remotesAddPushButton</tabstop>
  <tabstop>remotesRemovePushButton</tabstop>
  <tabstop>remoteListWidget</tabstop>
  <tabstop>sourceLineEdit</tabstop>
  <tabstop>destinationLineEdit</tabstop>
  <tabstop>browseTreeWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
